lib/entities/scheduled-action.ts

```diff
@@ -49,6 +49,7 @@ export type ScheduledActionProps = {
   environment?: { sys: MetaLinkProps }
   scheduledFor: {
     datetime: ISO8601Timestamp
+    timezone?: string
```

Comment:
Any chance we can add some /** JSDoc **/ here describing what we are expecting? See [contentful/types](https://github.com/contentful/content-stack/blob/master/packages/types/lib/types.ts#L6-L17) for inspiration

Reply:
Thank you!